packages/components/src/components/text/text.jsx

```diff
@@ -2,14 +2,13 @@ import classNames from 'classnames';
 import React from 'react';
 import PropTypes from 'prop-types';
 
-const Text = ({ children, size, color, align, weight, line_height, as, className, ...props }) => {
-    const style = {
+const Text = ({ children, size, color, align, weight, line_height, as, className, styles, ...props }) => {
```

Comment:
Why do we need styles?

Reply:
Actually, there are some exceptional cases and as per discussion with @zaki-binary  we decided to pass a style prop to handle them. For example, in this case, the default value for color makes problem and change the color of the text which had been 'inherit' and we had to change the default value of `font-color` to `inherit` in text component, but after discussion, we decided to add styles.